Simplenote/Classes/SPSettingsViewController+Extensions.swift

```diff
@@ -44,6 +44,11 @@ extension SPSettingsViewController {
         SPAppDelegate.shared().simperium.preferencesObject().isActiveSubscriber
     }
 
+    @objc
+    var wasSustainer: Bool {
+        SPAppDelegate.shared().simperium.preferencesObject().was_sustainer == true
+    }
```

Comment:
Nit: we can probably combine `isActiveSustainer` and `wasSustainer` into a single property?

```
var showsSustainerSwitch: Bool {
   let preferences = SPAppDelegate.shared().simperium.preferencesObject()
   return preferences.isActiveSubscriber || preferences.was_sustainer
}
```

?